modules/sdk/gradle-plugins-workspace/src/gradleTest/targetPlatformVersion/build.gradle

```diff
@@ -0,0 +1,25 @@
+allprojects {
```

Comment:
Please follow [these guidelines](https://github.com/liferay/liferay-portal/blob/master/modules/README.markdown#gradle-files), `task ...` should come before `allprojects`.